### DISCUSSION TITLE
Alpine LXC Nextcloud Login Creditentials will not display and not able to login.

### DISCUSSION THREAD

**Discussion by enter360** [tteck/Proxmox#2764] [Help Needed] [answered]
What is the method to login to the LXC container when the wget bash does not work ? I have created LXC using the script. Then pasted the same script into the LXC terminal. I get the error that the login incorrect. I restart the container and repeat the steps. Nothing changes. 

```
 Welcome to Alpine Linux 3.19
Kernel 6.5.13-1-pve on an x86_64 (/dev/tty1)

alpine-nextcloud login: bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ct/alpine-nextcloud.sh)"
Password: 
Login incorrect
alpine-nextcloud login: 
```

**Comment by tteck** [comment, marked as answer]
You type `root`

**Reply by enter360** [reply]
Thanks for the reply. It was such an obvious answer I didn't think to do it.
